python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,233 @@ class TabConfigView(TabDefault):
+    ----------
+    model : `Model`
+        Model class.
     """
 
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        # Labels
+        self._label_file_configuration = create_label()
+        self._label_file_version = create_label()
+        self._label_file_control_parameters = create_label()
+        self._label_file_lut_parameters = create_label()
+
+        self._label_power_warning_motor = create_label()
+        self._label_power_fault_motor = create_label()
+        self._label_power_threshold_motor = create_label()
+
+        self._label_power_warning_communication = create_label()
+        self._label_power_fault_communication = create_label()
+        self._label_power_threshold_communication = create_label()
+
+        self._label_in_position_axial = create_label()
+        self._label_in_position_tangent = create_label()
+        self._label_in_position_sample = create_label()
+
+        self._label_timeout_sal = create_label()
+        self._label_timeout_crio = create_label()
+        self._label_timeout_ilc = create_label()
+
+        self._label_misc_range_angle = create_label()
+        self._label_misc_diff_enabled = create_label()
+
+        self._label_misc_range_temperature = create_label()
+
+        # Internal widget and layout
+        self._set_widget_and_layout()
+
+        # Set the callback of signal
+        self._set_signal_config(self.model.signal_config)
+
+    def _set_widget_and_layout(self):
+        """Set the widget and layout."""
+
+        widget = self.widget()
+        widget.setLayout(self._create_layout())
+        self.setWidget(self.set_widget_scrollable(widget, is_resizable=True))
+
+    def _create_layout(
+        self, point_size_section=12, point_size_subsection=11, space_between_sections=30
+    ):
+        """Create the layout.
+
+        Parameters
+        ----------
+        point_size_section : `int`, optional
+            Point size of the section. (the default is 12)
+        point_size_subsection : `int`, optional
+            Point size of the subsection. (the default is 11)
+        space_between_sections : `int`, optional
+            Space between the sections. (the default is 30)
+
+        Returns
+        -------
+        layout : `PySide2.QtWidgets.QVBoxLayout`
+            Layout.
+        """
+
+        layout = QVBoxLayout()
+
+        label_file_version = create_label(
+            name="File Paths and Version", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_file_version)
+        layout.addWidget(self._label_file_configuration)
+        layout.addWidget(self._label_file_version)
+        layout.addWidget(self._label_file_control_parameters)
+        layout.addWidget(self._label_file_lut_parameters)
+
+        layout.addSpacing(space_between_sections)
+
+        label_power_supply = create_label(
+            name="Power Supply Monitoring", point_size=point_size_section, is_bold=True
+        )
```

Comment:
You are asking for trouble with this idea to separate logic groups using the different font sizes. Please look on e.g. QGroupBox for better ways to put related items visually together.

Reply:
Fixed in the commit: [29ca788](https://github.com/lsst-ts/ts_m2gui/pull/6/commits/29ca788f3c2b709d92963b35534482e61bb71a11). Thanks!